Changelog.md

```diff
@@ -31,39 +31,59 @@
   require an additional `MonadAsync` constraint. Several other
   functions that used these functions also now require the additional
   constraint.
-* Remove `Applicative` instance of folds. Please use `teeWith` or the `Tee`
-  type as an alternative to Fold applicative. To convert existing code:
 
-  ```
-    avg = (/) <$> Fold.sum <*> Fold.length
-  ```
+Removed `Applicative` instance of folds. Please use `teeWith` or the `Tee`
```

Comment:
This is not a bullet point anymore, what does this come under?

Reply:
Just a quick hack. Because when it is a bullet point, all the bullets have too much spacing between them, because this bullet is a para with blank lines. So I just separated it.

Reply:
Sure, it works I guess. There need not be a strict structure as long as the information is conveyed.